README.md

```diff
@@ -4,26 +4,45 @@
 
 > Flagsmith allows you to manage feature flags and remote config across multiple projects, environments and organisations.
 
-The SDK for Android and Java applications for [https://www.flagsmith.com/](https://www.flagsmith.com/).
+The SDK for Android and Java applications
+for [https://www.flagsmith.com/](https://www.flagsmith.com/).
 
 ## Adding to your project
 
-For full documentation visit [https://docs.flagsmith.com/clients/java/](https://docs.flagsmith.com/clients/java/)
+For full documentation
+visit [https://docs.flagsmith.com/clients/java/](https://docs.flagsmith.com/clients/java/)
 
 ## Contributing
 
-Please read [CONTRIBUTING.md](https://gist.github.com/kyle-ssg/c36a03aebe492e45cbd3eefb21cb0486) for details on our code of conduct, and the process for submitting pull requests
+Please read [CONTRIBUTING.md](https://gist.github.com/kyle-ssg/c36a03aebe492e45cbd3eefb21cb0486) for
+details on our code of conduct, and the process for submitting pull requests
 
 ## Getting Help
 
-If you encounter a bug or feature request we would like to hear about it. Before you submit an issue please search existing issues in order to prevent duplicates.
+If you encounter a bug or feature request we would like to hear about it. Before you submit an issue
+please search existing issues in order to prevent duplicates.
 
 ## Get in touch
 
-If you have any questions about our projects you can email <a href="mailto:support@flagsmith.com">support@flagsmith.com</a>.
+If you have any questions about our projects you can email <a href="mailto:support@flagsmith.com">
+support@flagsmith.com</a>.
 
 ## Useful links
 
 [Website](https://www.flagsmith.com/)
 
 [Documentation](https://docs.flagsmith.com/)
+
+## Code Style formatting
+
+We are using [Google Java Style](https://github.com/google/google-java-format) for code formatting.
+To add it to IntelliJ go to your IDE's settings and select the Plugins category. Click the
```

Comment:
We need to make this possible outside of IntelliJ. Also, can we add it to the github workflow?